intelmq/bots/parsers/cznic/parser_proki.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+import json
+
+from intelmq.lib import utils
+from intelmq.lib.bot import ParserBot
+
+
+class CZNICProkiParserBot(ParserBot):
+
+    recover_line = ParserBot.recover_line_json
+
+    def parse(self, report):
+        raw_report = utils.base64_decode(report.get("raw"))
+        events = json.loads(raw_report).get("data")
+        for line in events:
+            yield line
+
+    def parse_line(self, line, report):
+        event = self.new_event(report)
+
+        # json keys map 1:1 to harmonization fields
+        for field in line:
+            if field == "feed.name":
```

Comment:
The similar problem is also with `time.observation` field. 

Parsing fails with exception:
```
Traceback (most recent call last):
  File "/opt/dev_intelmq/intelmq/lib/bot.py", line 978, in process
    events = list(filter(bool, value))
  File "/opt/dev_intelmq/intelmq/bots/parsers/cznic/parser_proki.py", line 23, in parse_line
    event.add(field, line.get(field))
  File "/opt/dev_intelmq/intelmq/lib/message.py", line 224, in add
    raise exceptions.KeyExists(key)
intelmq.lib.exceptions.KeyExists: key 'time.observation' already exists
```

Reply:
Is this issue resolved? (at least the tests are ok)

Reply:
Is this issue resolved? (at least the tests are ok)